### DISCUSSION TITLE
New home, who dis? :new: :house_with_garden:

### DISCUSSION THREAD

**Discussion by queenofcorgis1** [community/community#27492] [New to GitHub]
Hey all - hope you’re making yourself comfortable in our new home, we’re glad to be here and settling in on Discussions. One of the first things you may have noticed is that our category has been updated. Just like painting the walls of your new apartment - we wanted to give a fresh update to our category as well. New to GitHub encompasses GitHub Pages, How to Use Git and GitHub, and Account Questions. 

And what better way for those new to GitHub to get more acquainted with the platform than by being in the platform itself. Any questions about Discussions or this new category ask them below! 

🌟**Community Superstars**: **Let’s give a standing ovation to these fine community folks for jumping right in on Discussions…**
* @mosesalfred is helping get the party started with [their question](https://github.com/orgs/community/discussions/27268) on errors after cloning a project from the repository. 
* @xqwtxon’s [comment](https://github.com/orgs/community/discussions/27407#discussioncomment-3266171) with a helpful visual graphic - makes way for all different learning styles! 
* @irod22 for a [great beginner question](https://github.com/orgs/community/discussions/27243) and @GAbdulWahab for solving it - nice work! 
* 13 comments and counting on @praveenab131’s post - thanks for posting and thanks to those who have made it so lively already! [Check out the action](https://github.com/orgs/community/discussions/27289) and get in on it yourself. 

🗯️ **Discussions Resources to Get You Started:** 
* 🗣️ [AMA: Community Powered by Discussions](https://github.com/orgs/community/discussions/27346): The Community Engagement team will be answering your questions about our move from 9am PST on July 28th - 9am PST July 29th! 
* ✅ [New User Checklist](https://github.com/orgs/community/discussions/27367): Begin your journey to become a GitHub expert or brush up on core skills! 
* 👋 [Welcome Center](https://github.com/orgs/community/discussions/categories/welcome-center): Learn how to navigate our new space, get updates, and introduce yourself. 

👀 **Looking for more to do?** 
* Continue the Discussions bonanza with [this article](https://github.blog/2022-07-20-heres-how-academic-research-is-shaping-github-discussions/) from @evi-liu about how academic research is shaping Discussions 🎒. 
* [Tell us](https://github.com/community/community/discussions/27366) how you use GitHub @ work and share any tips! 

Take some time to settle in and explore, we’re excited for this next chapter in GitHub Community and even more thrilled you’re along for the ride with us.

**Comment by xRodzMC** [comment]
We glad to say we're here to help.

**Comment by jfullstackdev** [comment]
hello there @queenofcorgis

**Reply by queenofcorgis1** [reply]
Hi @jdevstatic - nice to see you 👋 

How are you liking Discussions so far?

**Reply by jfullstackdev** [reply]
Discourse will have a competitor for sure 😅, although the original idea behind GitHub Discussion is for developers to discuss things

it's almost the same thing that happened when GitHub Pages was released, it was intended for documentation of a software, but more and more users joined GitHub just for that feature alone and was used to build their own sites rather than being used for documentation

all in all, they are very good innovations

**Reply by queenofcorgis1** [reply]
Glad to hear you think so 👍 Thanks for all the participation and feedback so far - we ❤️ it!

**Comment by melvincwng** [comment]
Hello @queenofcorgis , nice to see a consolidated platform for GitHub Community Forums & GitHub Discussions 👍🏻. 

Am glad to see the different initiatives by the GitHub Staff to build a more inclusive and collaborative community. Great work!

Off-topic: Corgis are super cute 🐶! Great choice for a username 😜

**Reply by queenofcorgis1** [reply]
Hi @melvincwng! Thanks for helping out so much in the community - it is awesome to see you diving in! If you have any initiatives you'd like to see here - I'm all ears! 

And agreed - corgis are quite fantastic, thank you!

**Reply by melvincwng** [reply]
@queenofcorgis , no problem - the pleasure is mine in helping our fellow members of the GitHub Community :)

Sure thing, will let you know if I have any ideas/initiatives that can be added to this platform!

**Reply by melvincwng** [reply]
@queenofcorgis Hi Lili, just a quick question.

I've noticed that the 'Discussions Answered' count is missing in all GitHub profiles. I was wondering this is a known issue by the GitHub Team? Or would I have to report it as a bug to GitHub Bug Report?

There was a previous thread created 2 days ago regarding this - for your reference: https://github.com/community/community/discussions/27346#discussioncomment-3281025 

Do let me know if you have any inputs, thanks ^^

**Comment by airtower-luna** [comment]
So, about corgis… Is the one in the profile picture yours? Maybe one of yours? :dog:

**Reply by queenofcorgis1** [reply]
I wish!! Unfortunately, my apartment doesn't allow pets luckily my neighborhood seems to be full of them, I see one almost every day.

**Reply by jfullstackdev** [reply]
how about cats ? I appreciate both haha we're renting too but we still manage to have one cat 😅

**Reply by queenofcorgis1** [reply]
No pets of any kind - but lots of plants as my windows face south so I get great light 🌞 🪴

**Reply by jfullstackdev** [reply]
nice one, that direct sunlight in temperate zone is very important, while in the tropics, we still need and love that, but you want to hide in the shade 😅 worst either too much UV or heatstroke particularly in the months of April and May 😅 but we are attuned to it haha that's the nice thing

**Reply by airtower-luna** [reply]
Well, I'm glad you get to see them at least! Reminds me of the neighbor's fluffy cat who said hi to me yesterday. :smile_cat:
